robot/basestation/templates/AutomaticControls.html

```diff
@@ -0,0 +1,111 @@
+<!-- Automatic controls section -->
+<table class="table-grp" style="display: block" id="ArmcontrolsOFF" align="center">
+  <tr> <!-- first row -->
+    <td></td>
+    <td valign="bottom">
+      <a href="javascript:void(0)" id="click_btn_pitch_up">
+        <button type="button" class="control-btn">
+          <strong>Pitch<br>Up</strong>
```

Comment:
Yeah that's not my code though (you know because I would never use html comments lol), it's just something I moved. Ill see what I can do once I have access to a computer (writing this on my phone)